### DISCUSSION TITLE
A clerical mistake in simplified-Chinese

### DISCUSSION THREAD

**Discussion by Xeler-ync** [mdn/mdn-community#331] [Localization]
I have some confusion about a sentence in the simplified-Chinese page [正负自由空间][正负自由空间] in the chapter [PositiveAndNegativeFreeSpace][PositiveAndNegativeFreeSpace] of the MDN document,. In the passages of "备注", "为了不翻译**花**了" is a bit strange in this passage. Considering the homonyms in Chinese, I guess it should be "为了不翻译**化**了", which means "to avoid translation". The pronunciation of "花" and "化" in Chinese is familiar, which may make the translator make mistakes when inputting. But it may be my misunderstanding. Please let me know if so :D
Thank you for your attention to this matter

[正负自由空间]: https://developer.mozilla.org/zh-CN/docs/Web/CSS/CSS_Flexible_Box_Layout/Controlling_Ratios_of_Flex_Items_Along_the_Main_Ax#%E6%AD%A3%E8%B4%9F%E8%87%AA%E7%94%B1%E7%A9%BA%E9%97%B4
[PositiveAndNegativeFreeSpace]: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Controlling_Ratios_of_Flex_Items_Along_the_Main_Ax

**Comment by yin1999** [comment]
Hi @Xeler-ync. We should remove those notes and update this document. Could you please create an issue in translated-content. You could click the "content issue" link under [分配空间的其他方式](https://developer.mozilla.org/zh-CN/docs/Web/CSS/CSS_Flexible_Box_Layout/Controlling_Ratios_of_Flex_Items_Along_the_Main_Ax#%E5%88%86%E9%85%8D%E7%A9%BA%E9%97%B4%E7%9A%84%E5%85%B6%E4%BB%96%E6%96%B9%E5%BC%8F) to create this.

![image](https://user-images.githubusercontent.com/15844309/219842714-62a2fe70-5543-48ea-b78f-5388c46ffeed.png)

**Reply by Xeler-ync** [reply]
I'm sorry, this is the first time I give an issue for MDN Web Docs and I'm not familiar with the process. I will go to "content issue" create this 😵‍💫

**Comment by pransh15** [comment]
Closing this since it was answered.
